### ISSUE TITLE
Sitch: Linux FAIL :(

### ISSUE THREAD

**Issue by im-coder-lg** [not-nef/txt2#51]
> im fed up, ugh

Tried installing on Linux, and plot twist, pywin32 and tkinterweb failed to install!

**Comment by im-coder-lg** [comment]
fed up with me

**Comment by im-coder-lg** [comment]
*correction

**Comment by not-nef** [comment]
epic

**Comment by not-nef** [comment]
pywin can be circumvented with a few lines of code in `generatesize.py`, but tkinterweb is absolutely essential fir markdown preview.

**Comment by not-nef** [comment]
What wheelfile? You tryna force install it or what

**Comment by im-coder-lg** [comment]
Yep, figured it out. Maybe I did a mistake while putting the command or summat, that's fixed now.

**Comment by im-coder-lg** [comment]
What's the code for circumventing pywin?

I better add the os module and circumvent that during next week Sunday. Calendar is booked with projects for me.

**Comment by not-nef** [comment]
Ah wait the code is already built so it only relies on pywin when on windows, you should be able to run it if you have tkinterweb

**Comment by im-coder-lg** [comment]
I am currently install PIL's ImageTK part from the Ubuntu Universe repos, I will be back when I fix this. I have to document this too.

**Comment by im-coder-lg** [comment]
*installing

**Comment by im-coder-lg** [comment]
You used Python 3.10, which has the removesuffix attribute for str. THat is causing much trouble, so I am install Anaconda with Py3.9

**Comment by im-coder-lg** [comment]
*installing

WHAT IS WRONG WITH ME

**Comment by not-nef** [comment]
> You used Python 3.10, which has the removesuffix attribute for str. THat is causing much trouble, so I am install Anaconda with Py3.9

what

**Comment by im-coder-lg** [comment]
I am done for today. It's nearing my bedtime, and I got to do a bunch of work at school. The teachers made me monitor for class(not _prefect_ since there is no such thing here for me) and I got a bunch of collection of book work. Added to that is a Social project, and added to _that_ is 6 days of sleep loss. So I have to prepare for the weird war ahead, with my birthday coming soon(I'm gonna be 14), I need to be ready for the work. So, I will try getting circumvention for this, possibly by using Darwin and Linux(Thanks Steve Jobs and Linus Torvalds for making the platform name easy to remember) and fixing that. Then, we could make macOS and Linux builds.

**Comment by im-coder-lg** [comment]
> > You used Python 3.10, which has the removesuffix attribute for str. THat is causing much trouble, so I am install Anaconda with Py3.9
> 
> what

Forgive me for my lack of grammar(and brains) nef. What I meant was, you used Python3.9+(remember, plus in the end), which had removesuffix attribute for the str thing, which is causing errors for Python 3.8 and below. So I had to get Anaconda. And just now I remembered, I have the asdf package manager on my computer. Idiot me forgot that it supports Node, Golang, and even Python(wait does it)! Man, I will try tomorrow or Sunday, as I said before, the project.
